eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState.hs

```diff
@@ -1403,25 +1346,18 @@ resolveActiveIncrementalPtrs isActive ptrMap (IStake credStake ptrStake) =
             then Map.insertWith (<>) cred coin ans
             else ans
 
--- | Aggregate active stake by merging two maps. The triple map from the UnifiedMap, and the IncrementalStake
---   Only keep the active stake. Active can be determined if there is a (SJust deleg) in the Triple.
---   This is step2 =  aggregate (dom activeDelegs ◁ rewards) step1
+-- | Aggregate active stake by merging two maps. The triple map from the
+--   UnifiedMap, and the IncrementalStake Only keep the active stake. Active can
+--   be determined if there is a (SJust deleg) in the Triple.  This is step2 =
+--   aggregate (dom activeDelegs ◁ rewards) step1
 aggregateActiveStake :: Ord k => Map k (Triple crypto) -> Map k Coin -> Map k Coin
 aggregateActiveStake tripmap incremental =
   Map.mergeWithKey
     -- How to merge the ranges of the two maps where they have a common key. Below
     -- 'coin1' and 'coin2' have the same key, '_k', and the stake is active if the delegation is SJust
-    ( \_k triple coin2 ->
-        case triple of
-          (Triple (SJust coin1) _ (SJust _)) -> Just (coin1 <> coin2)
-          _ -> Nothing
-    )
+    (\_k trip coin2 -> (<> coin2) <$> UM.tripDelegatedReward trip)
     -- what to do when a key appears just in 'tripmap', we only add the coin if the key is active
-    ( \mp ->
-        let p _key (Triple (SJust c) _ (SJust _)) = Just c
-            p _ _ = Nothing
-         in filterMaybe p mp
-    )
+    (Map.mapMaybeWithKey (\_ -> UM.tripDelegatedReward))
```

Comment:
Maybe add a comment that tripDelegatedReward returns (Just coin) if there is a reward.
Or maybe change the name so that that fact is implicit in the name.

Reply:
Will change to: `tripRewardActiveDelegation`